scripts/detection/faster_rcnn/train_faster_rcnn.py

```diff
@@ -114,6 +114,8 @@ def parse_args():
     parser.add_argument('--kv-store', type=str, default='nccl',
                         help='KV store options. local, device, nccl, dist_sync, dist_device_sync, '
                              'dist_async are available.')
+    parser.add_argument('--dtype', type=str, default='float32',
```

Comment:
Can u add some documentation about the difference between AMP and float16? or we could merge float16 and AMP into one option?

Reply:
Merged AMP and float16 into one option and also added a small comments about float16.